sonar/help/templates/help/page_wiki_search.html

```diff
@@ -0,0 +1,61 @@
+{# -*- coding: utf-8 -*-
+  Swiss Open Access Repository
+  Copyright (C) 2021 RERO
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published by
+  the Free Software Foundation, version 3 of the License.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#}
+
+{% from 'bootstrap/form.html' import render_form_row %}
+
+{% extends 'help/page_wiki_base.html' %}
+{% set active_page = "search" %}
+
+{% block content %}
+<main class="row mt-4 wiki-page">
+    <article class="col-md-10 offset-md-1">
+        <header>
+            <div class="pb-3">
+                {{ results | length }} {{ ngettext('result', 'results', results | length) }}
+                <a href="{{ url_for('help.search', view=view, q='*') }}" class="btn btn-sm btn-outline-primary ml-2">{{ _("All languages") }}</a>
+                </div>
+        </header>
+        <article>
+            <ul class="list-group list-group-flush">
+                {%- for result in results -%}
+                <li class="list-group-item">
+                    <article class="d-lg-flex justify-content-lg-between">
+                        <article class="mb-2">
+                            <h5 class="m-0">
+                                <a class="mr-2" href="{{ url_for('help.page', view=view, url=result.url) }}">
```

Comment:
I imagine this is the only modification done form flask-wiki. We can perhaps create a link factory for this?